src/components/Post/GovernanceSideBar/Motions/VoteMotion.tsx

```diff
@@ -62,23 +62,24 @@ const VoteMotion = ({ accounts, address, className, getAccounts, motionId, motio
 	const [voteCount, setVoteCount] = useState<number>(0);
 
 	useEffect(() => {
-		if (!api) {
-			return;
-		}
-
-		if (!apiReady) {
-			return;
-		}
-
+		if (!api || !apiReady) return;
 		if (accounts.length === 0) {
 			getAccounts();
 		}
 
-		api.query?.council?.members().then((memberAccounts) => {
-			const members = memberAccounts.map((member) => member.toString());
-			setCurrentCouncil(members.filter((member) => !!member) as string[]);
-		});
-
+		if (api?.query?.council) {
+			api.query.council
+				.members()
+				.then((memberAccounts) => {
+					const members = memberAccounts.map((member) => member.toString());
+					setCurrentCouncil(members.filter((member) => !!member) as string[]);
+				})
+				.catch((error) => {
+					console.error('Error fetching council members:', error);
+				});
+		} else {
+			console.error('Council pallet is not available on this network.');
+		}
 		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
remove 		// eslint-disable-next-line react-hooks/exhaustive-deps

Reply:
it gives warning if i remove this eslint disable comment